include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,8 +917,8 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+1), additional_data);
-      typedef MatrixFreeOperators::MassOperator<dim, fe_degree, 1, Number> MatrixType;
+                          QGauss<1>(fe_degree+2), additional_data);
+      typedef MatrixFreeOperators::MassOperator<dim, fe_degree, fe_degree+2, 1, Number> MatrixType;
```

Comment:
I see, I thought that the same quadrature formula would have been used in the right hand side and the mass matrix. (It says so in the general documentation of the old project() function: "quadrature The quadrature formula to be used for assembling the mass matrix". It seems reasonable to only go through one path, I agree. It would be good to document this behavior clearly in the header.